### PR TITLE
AR-1566 AR-1698 Add support for running the PUI under a URL prefix

### PR DESCRIPTION
**Pull request by marktriggs**
Fun on a bun!

### CODE REVIEW

File: public/app/assets/stylesheets/archivesspace/aspace.scss [hunk 1]
```diff
@@ -382,7 +382,7 @@ a:hover {
 
 
 /* .logo { */
-/*   background-image: url('../img/Aspace-logo-transparent.png'); */
+/*   background-image: asset_url('Aspace-logo-transparent.png'); */
```
Comment (lmcglohon) [inline public/app/assets/stylesheets/archivesspace/aspace.scss]: @marktriggs Shouldn't this be asset-url and not asset_url?

File: public/app/assets/stylesheets/archivesspace/aspace.scss [hunk 2]
```diff
@@ -493,7 +493,7 @@ h5 {
   border-bottom: 1px solid;
 }
 li.remove {
-  background-image: url("../img/remove.png");
+  background-image: asset_url("remove.png");
```
Comment (lmcglohon) [inline public/app/assets/stylesheets/archivesspace/aspace.scss]: @marktriggs Shouldn't this be asset-url and not asset_url?

File: public/app/views/pdf/_header.html.erb [hunk 3]
```diff
@@ -198,4 +198,4 @@
     </head>
     <body>
 
-        <div class="logo"><%= image_tag("/img/Aspace-logo.png") %></div>
+        <div class="logo"><%= image_tag(asset_url("Aspace-logo.png")) %></div>
```
Comment (lmcglohon) [inline public/app/views/pdf/_header.html.erb]: @marktriggs Shouldn't this be asset-url and not asset_url?

File: public/app/views/shared/_header.html.erb [hunk 4]
```diff
@@ -2,6 +2,6 @@
   <div class="row">
     <div class="col-sm-9 h1"><% unless request.original_fullpath == '/' %>
         <a title="<%=t('brand.title_link_text') %>" href="<%= AppConfig[:public_url] %>"><% end %> <%= t('brand.title') %><% unless request.original_fullpath == '/' %></a><% end %> </div>
-    <div class="col-sm-3 hidden-xs"><img class="logo" src="<%= AppConfig[:pui_branding_img] %>" alt="" /></div>
+    <div class="col-sm-3 hidden-xs"><img class="logo" src="<%= asset_url(AppConfig[:pui_branding_img]) %>" alt="" /></div>
```
Comment (lmcglohon) [inline public/app/views/shared/_header.html.erb]: @marktriggs Shouldn't this be asset-url and not asset_url?

**Comment by marktriggs** [comment]
Thanks @lmcglohon!

The `.scss` files seem to allow either asset-url or asset_url, but it turns out to be moot because when I dug around I couldn't find anywhere that actually used that class (or the image).  I've removed both now.

The other uses of `asset_url` are in Ruby-land are should be right (that's hitting `asset_url` from http://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html)

**Pull request merged**